### ISSUE TITLE
Add more content types as subcontent

### ISSUE THREAD

**Issue by otacke** [otacke/h5p-game-map#10]
Add more content types as subcontent.

Some content types seemed to cause trouble when used as subcontent, and I’ll have to add a workaround or the content type might need changing.

**Comment by otacke** [comment]
_Requested_
- Matching: Not now. Not actively maintained by H5P Group.
- Questionnaire: The currently released version 1.3.5 is not suitable as subcontent in a question-like context. It lacks the resetTask function for instance. The current development version 1.3.7 has that function, but is not working properly - and 1.3 could not be referenced anyway, beacuse of the API change. Will have to wait for the 1.4 version to be available.
- JigsawPuzzle: Could give bad UX due to own audio mute button. Would need API to delegate audio handling to parent content type.

**Comment by citrulluslanatus** [comment]
A very useful subcontent would also be a **combination** of text and image. At the moment I have to insert text into my images to achieve this, but that never really looks good and I lose all ability to customise that text through css later (and of course I can't change the text itself).

**Comment by otacke** [comment]
@citrulluslanatus https://github.com/otacke/h5p-game-map/issues/91

**Comment by citrulluslanatus** [comment]
Yes, that would do it :). Thanks! I hope you find the time and funding.

**Comment by otacke** [comment]
@citrulluslanatus Just going through open issues. Just wanted to let you know that the latest version of GameMap (1.4+) allows to add multiple exercises to a stage. Please visit https://www.olivertacke.de/labs/2025/02/10/gamemap-1-4-is-out/ for details.

**Comment by citrulluslanatus** [comment]
Hi. THANK YOU, that is very very useful. I just checked your post regarding all new features I 1.5 - this will open a lot of opportunities.
I use your game map regularly to find new ways for motivational teaching, e.g. here: https://grammatik.inueb.net
Looking forward for all future features to come :).

**Comment by otacke** [comment]
@citrulluslanatus That looks fantastic! You should have entered that content to the [H5P Awards 2025](https://h5p.org/the-second-annual-h5p-awards). Too late now, but maybe next year.

**Comment by citrulluslanatus** [comment]
@otacke I had briefly thought about it :) , but since it's not in English and some ideas don't work so well (although the new functions will help a lot there), not yet. But especially the individual paths and the integration of several content types per station is enormously cool for the application for educational content.

**Comment by otacke** [comment]
@citrulluslanatus Just think about it next year. The language is not relevant - you will see nominations in different languages shortly.

**Comment by vioannou** [comment]
@otacke I'm wondering if you could add "Speak the word" as a content type. This would make the map an even better tool for online language learning classes.
